File: Prak405/PRAK405-2210817210021-TRISNACAHYAPERMADI.c
#include <stdio.h>
#include <conio.h>
int main (void)
{
    int x, y, hasil, hasil1;
    scanf ("%i", &x);
    scanf ("%i", &y);
    for (int i = 0; i < x; i++){
        for (int j = i; j>=0; j--){
            printf ("(%i * %i)" , j+1, y);
            if (j > 0){
                printf (" + ");
            }
        }
        hasil += (i+1) * y;
        printf (" = %i\n", hasil);
        hasil1 += hasil;
    }
    printf ("%i", hasil1);
}

File: Prak405/PRAK405-2210817210021-TRISNACAHYAPERMADI.py
input_pertama,input_kedua = input ().split()
input_pertama = int (input_pertama)
input_kedua = int (input_kedua)
list_input = range (1, input_pertama +1)
data_list = list (list_input)
hasil_akhir = 0
for i in data_list :
    hasil = 0 
    hasil1 = 0
    for n in reversed ( range (i)) :
        if n == 0 :
            print (f"({n+1} * {input_kedua})", end="")
            hasil1 = (n+1) * input_kedua
            hasil += hasil1
        else :
            print (f"({n+1} * {input_kedua}) + ", end="")
            hasil1 = (n+1) * input_kedua
            hasil += hasil1
    print (f" = {hasil}")
    hasil_akhir += hasil
print (hasil_akhir)
    